WordPress/Tests/WP/I18nUnitTest.1.inc

```diff
@@ -151,5 +151,9 @@ really.
 EOD
 , 'my-slug' ); // OK.
 
-// @codingStandardsChangeSetting WordPress.WP.I18n text_domain null
```

Comment:
This line should not be removed. It **_has_** to be at the end of the file to reset the properties for the other test files, so it should come back below the newly added test cases.